tests/licensedcode/data/example_external_licenses/licenses_to_install1/gpl-1.0.LICENSE

```diff
@@ -0,0 +1,254 @@
+GNU GENERAL PUBLIC LICENSE
```

Comment:
Using something shorter and with a smaller file size for an example plugin may be better? This is going to be checked out 1000's of times ;)